subgraph/origin/subgraph.yaml

```diff
@@ -0,0 +1,29 @@
+specVersion: 0.0.2
+schema:
+  file: ./schema.graphql
+dataSources:
+  - kind: ethereum/contract
+    name: ERC20Gateway
+    network: dev
+    source:
+      address: "0xdbbb5c822ccfac3ea49e32215565c6aafb257a30"
+      abi: ERC20Gateway
+    mapping:
+      kind: ethereum/events
+      apiVersion: 0.0.3
+      language: wasm/assemblyscript
+      entities:
+        - DepositIntentDeclared
+        - GatewayProven
```

Comment:
entity names are incorrect. It should align with schema.graphql entity names

Reply:
Fixed.